src/main/scala/com/exasol/spark/util/Types.scala

```diff
@@ -177,10 +178,16 @@ object Types extends Logging {
     case BinaryType      => "CLOB"
     case DateType        => "DATE"
     case TimestampType   => "TIMESTAMP"
-    case _ =>
-      throw new IllegalArgumentException(s"Unsupported Spark data type $dataType!")
+    case _               => throw new IllegalArgumentException(getUnsupportedDataTypeErrorCode(dataType))
   }
 
+  private[this] def getUnsupportedDataTypeErrorCode(dataType: DataType): String =
+    ExaError
+      .messageBuilder("F-SEC-8")
+      .message("Unsupported Spark data type {{DATA_TYPE}} is not supported.", String.valueOf(dataType))
```

Comment:
Unsupported data type is not supported -> this sounds funny :)) Maybe rephrase?

Reply:
Typo from previous message, fixed it.